### ISSUE TITLE
About dataset

### ISSUE THREAD

**Issue by caoyu227** [tansor/VecRoad#12]
Hello,I just see the mask for testing dataset, so can you tell me where is the mask for training dataset, Thank you.

**Comment by tansor** [comment]
Training scripts have been released, please refer to train.py. Let me know if there are still questions.
